awx/main/management/commands/inventory_import.py

```diff
@@ -124,7 +126,11 @@ def get_path_to_ansible_inventory(self):
 
     def get_base_args(self):
         # get ansible-inventory absolute path for running in bubblewrap/proot, in Popen
-        bargs= [self.get_path_to_ansible_inventory(), '-i', self.source]
+        ansible_inventory_path = self.get_path_to_ansible_inventory()
+        bargs= [ansible_inventory_path, '-i', self.source]
+        ansible_version = _get_ansible_version(ansible_inventory_path[:-len('-inventory')])
+        if ansible_version != 'unknown' and Version(ansible_version) >= Version('2.5'):
+            bargs.extend(['--playbook-dir', self.source_dir])
```

Comment:
ugh, this is still technically blocked by

https://github.com/ansible/ansible/pull/51177

Yes, the flag existed in 2.5, but it didn't necessary work right. I don't know what use case we need that for.

We do kind of need to ask if they intend to backport that.

I'm of a mind to only set this if the version is 2.8 or over, and do this along with verbosity.

Reply:
The question has been asked in the Ansible link.